src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/CreditCardMapper.php

```diff
@@ -242,12 +244,43 @@ protected function expandCreditCardPaymentWithCredentialOnFile(
         $computopCredentialOnFileTransfer = (new ComputopCredentialOnFileTransfer())
             ->setType(
                 (new ComputopCredentialOnFileTypeTransfer())
-                    ->setUnscheduled(ComputopApiConfig::UNSCHEDULED_CUSTOMER_INITIATED_TRANSACTION)
+                    ->setUnscheduled(ComputopApiConfig::UNSCHEDULED_CUSTOMER_INITIATED_TRANSACTION),
             )
             ->setInitialPayment(true);
 
         $computopCreditCardPaymentTransfer->setCredentialOnFile($computopCredentialOnFileTransfer);
 
         return $computopCreditCardPaymentTransfer;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\AddressTransfer $addressTransfer
+     * @param \Generated\Shared\Transfer\ComputopConsumerTransfer $computopConsumerTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopConsumerTransfer
+     */
+    protected function mapAddressTransferToConsumerTransfer(
+        AddressTransfer $addressTransfer,
+        ComputopConsumerTransfer $computopConsumerTransfer
+    ): ComputopConsumerTransfer {
+        $computopConsumerTransfer->fromArray($addressTransfer->toArray(), true);
+        if ($computopConsumerTransfer->getSalutation()) {
+            $computopConsumerTransfer->setSalutation($this->getAcceptedSalutation($computopConsumerTransfer->getSalutation()));
+        }
+
+        return $computopConsumerTransfer;
+    }
+
+    /**
+     * @param string $salutation
+     *
+     * @return string
+     */
+    protected function getAcceptedSalutation(string $salutation): string
+    {
+        $salutationMap = $this->config->getSalutationMap();
+        $salutation = trim(str_replace('.', '', $salutation));
+
+        return $salutationMap[$salutation] ?? 'Mr';
```

Comment:
Please move `Mr` to a constant. Also, I think it makes sense to move it to config, e.g.
`getDefaultSalutation`

Reply:
Fixed